src/codegen/c.zig

```diff
@@ -1162,12 +1162,12 @@ fn genBody(f: *Function, body: []const Air.Inst.Index) error{ AnalysisFail, OutO
 
             .slice => try airSlice(f, inst),
 
-            .cmp_eq  => try airBinOp(f, inst, " == "),
+            .cmp_eq  => try airEquality(f, inst, true),
```

Comment:
true / false are almost always code smells. Passing the operation in is much clearer and you can easily derive the boolean inside the method call

Reply:
Thank you :) should be good now